source/beamline/SGM/SGMBypassLadder.h

```diff
@@ -0,0 +1,32 @@
+#ifndef SGMBYPASSLADDER_H
+#define SGMBYPASSLADDER_H
+
+#include "beamline/SGM/SGMLadderControl.h"
+
+#define SGMBYPASSLADDER_SLIT_SETPOINT 1260
+#define SGMBYPASSLADDER_SLIT_MIN 0
+#define SGMBYPASSLADDER_SLIT_MAX 1260
+
+#define SGMBYPASSLADDER_PASSTHROUGH_SETPOINT 12000
+#define SGMBYPASSLADDER_PASSTHROUGH_MIN 12000
```

Comment:
Are these min/max values supposed to be exactly the same as the setpoint? And is the SLIT the only one that is supposed to be different?